### ISSUE TITLE
[FEATURE REQUEST] email notification (patch included)

### ISSUE THREAD

**Issue by yoyoma2** [mag37/dockcheck#43]
This script is great. I just wanted to add email notification for my synology DSM and it works with a few lines.

[DSMemailnotification.txt](https://github.com/mag37/dockcheck/files/13232993/DSMemailnotification.txt)

It's quite specific to DSM emails however. How can I still benefit from the script's nice self update feature if email notifications don't make it in?

One line in the patch is a general fix to find regctl when running as a cron job (the script itself and regctl are not in the path but regctl is with the script. This line should make it in regardless of email notification.

**Comment by mag37** [comment]
Sorry for the late reply! The point about the regctl binary check is a good one, thank you for reporting it.
The email-notification is a good suggestion, thou I wont add it for now - mainly because its pretty niche and requires external functionality (ssmtp).
If you want notifications there's the spinoff-project [palleri/DCW](https://github.com/Palleri/DCW) that handles all the big notification protocols.

I'll leave this open for now for consideration, and until I've added the regctl-fix. 
Thank you again.

**Comment by yoyoma2** [comment]
Agreed it's niche to synology DSM. That's why I didn't make a hopeless pull request. I did see that brother project and it's overkill and seems complicated to set up. This is working great and I got docker update notification emails from my NAS this week.

Glad that the regctl-fix might make it in.

Idea:
Create a file called notify_DSM.sh.sample (it contains send_notification() that works with synology DSM)
If file notify.sh exists it is sourced and send_notification() is called in the right place. (no -m needed)
To use it, DSM users copy/tweak notify_DSM.sh.sample to notify.sh and the feature works.

Other future users submit pull requests adding files notify_XYZ.sh.sample for their XYZ platform.
Users can still benefit from this script's nice self-update feature.

Note:
I explain poorly. I could make a pull request showing this if it makes sense to you.
